aspnetcore/fundamentals/minimal-apis/7.0-samples/WebMinJson/Program.cs

```diff
@@ -1,4 +1,7 @@
-#define First
+#define writeasjsonasyncwithoptions
+// First Second writeasjsonasync autojsondeserialization
+// readfromjsonasync confighttpjsonoptions jsonoptions writeasjsonasyncwithoptions
+// readfromjsonasyncwithoptions resultsjsonwithoptions
```

Comment:
Are these comments for SEO or something?

Reply:
Those are the various symbols that may be defined to enable compilation of different snippets in the file. It's just a copy/paste convenience for adding to the `#define` directive.